docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,82 +711,85 @@ To read and write a value of the Bluetooth descriptor:
 
    descriptor.writeValue(newValue, writeSuccess, writeFail);
    ```
-<!-- TODO #04
+<!-- END #05 -->
+
+<!-- TODO #06
       - write about getAttMtu()
       - write about requestAttMtuChange()
  -->
 ### Accessing the ATT MTU of connected device
-<!-- END #04 -->
+<!-- END #06 -->
 
-<!-- TODO #05
+<!-- TODO #07
       - write about addAttMtuChangeListener()
       - write about removeAttMtuChangeListener()
  -->
 ### Receiving Notifications on ATT MTU changes
-<!-- END #05 -->
+<!-- END #07 -->
 
 
 ## Managing the local GATT server
 
-<!-- TODO #06
+<!-- TODO #08
       - write about start()
  -->
 ### Start the server
-<!-- END #06 -->
+<!-- END #08 -->
 
-<!-- TODO #07
+<!-- TODO #09
       - write about stop()
  -->
 ### Stop the server
-<!-- END #07 -->
+<!-- END #09 -->
 
-<!-- TODO #08
+<!-- TODO #10
       - write about addConnectStateChangeListener()
       - write about removeConnectStateChangeListener()
  -->
 ### Receiving notifications on server's Connection State changes
-<!-- END #08 -->
+<!-- END #10 -->
 
-<!-- TODO #09
+<!-- TODO #11
       - write about getConnectionMtu()
  -->
 ### Accessing connection's ATT MTU
-<!-- END #09 -->
+<!-- END #11 -->
 
-<!-- TODO #10
+<!-- TODO #12
       - write about registerService()
  -->
 ### Registering services
-<!-- END #10 -->
+<!-- END #12 -->
 
-<!-- TODO #11
+<!-- TODO #13
       - write about unregisterAllServices()
       - write about BluetoothGATTServerService::unregister()
  -->
 ### Unregistering services
-<!-- END #11 -->
+<!-- END #13 -->
 
-<!-- TODO #12
-      - write about setReadValueRequestCallback() for characteristic
-      - write about setWriteValueRequestCallback() for characteristic
+<!-- TODO #14
+      - write about notifyAboutValueChange()
  -->
-### Managing characteristics of server service
-<!-- END #12 -->
+### Sending notifications to the clients about characteristic's value changes
```

Comment:
I'd change the order of words in this sentence: "Sending notifications about characteristic's value changes to the clients"